docs/scos/dev/glue-api-guides/202311.0/create-glue-api-applications.md

```diff
@@ -305,7 +305,7 @@ Each method in `CustomApiApplication` represents a step in the API application r
 
 This application extends `RequestFlowAwareApiApplication`, which means that this API application follows the default Glue workflow. This is beneficial because it lets you make use the most of the Spryker conventions and features that wire into the request flow.
 
-If your API uses its own workflow, you can opt for extending `RequestFlowAgnosticApiApplication`. This kind of application has everything—a separate set of application plugins, boot, and run methods—but not the request flow actions. The application that follows the `RequestFlowAgnosticApiApplication` extension is the old Glue storefront API application. It is request-agnostic and creates and follows its own request flow. Here is an example of the application:
+If your API uses its own workflow, you can opt for extending `RequestFlowAgnosticApiApplication`. This kind of application has everything—a separate set of application plugins, boot, and run methods—but not the request flow actions. The application that follows the `RequestFlowAgnosticApiApplication` extension is the Legacy Glue API application. It is request-agnostic and creates and follows its own request flow. Here is an example of the application:
```

Comment:
what is "RequestFlowAgnosticApiApplication" ? class / template/ reference? 

please provide useful scenarios where standard request handling mechanisms do not fit the specific needs of the application or where there is a need to integrate with legacy systems.